### PR TITLE
[TTAHUB-1839] Rename `pocId` to `pocIds`

### PR DESCRIPTION
**Pull request by nvms**
## Description of change

`pocId` is now an array of integers, so the name should be updated to reflect that.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1839

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/TrainingReportForm/__tests__/index.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ describe('TrainingReportForm', () => {
       },
       collaboratorIds: [1],
       ownerId: 1,
-      pocId: 1,
+      pocIds: 1,
```
Comment (thewatermethod) [inline frontend/src/pages/TrainingReportForm/__tests__/index.js:133]: should this be converted to an array?

File: docs/openapi/paths/event/index.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ post:
           properties:
             ownerId:
               type: integer
-            pocId:
+            pocIds:
```
Comment (thewatermethod) [inline docs/openapi/paths/event/index.yaml:14]: this should be updated?

Comment (thewatermethod) [reply docs/openapi/paths/event/index.yaml:14]: and the preceding openapi stuff

**Comment by nvms** [comment]
I merged main and had to fix a conflict in a file with CRLF endings. I guess my editor overwrites these with LF transparently, so there are a few files that might be harder to diff than others, sorry!

**Comment by thewatermethod** [comment]
May want to work out sequencing with #1619  before merging

File: src/migrations/20230626183058-alter-poc-col.js
Comment (GarrettEHill) [inline src/migrations/20230626183058-alter-poc-col.js:1]: Why not use the rename column command? queryInterface.renameColumn('my_some_table', 'totoId', 'toto_id');

Comment (nvms) [reply src/migrations/20230626183058-alter-poc-col.js:1]: good call: 1d5563a3c9aca54d5283b8b14f22d62460452fc4

**Review by thewatermethod — APPROVED** [review]
Looks good to me, CRUD still worked on events and sessions

**Pull request merged**